src/Admin/VMware.ScriptRuntimeService.Setup/Program.cs

```diff
@@ -19,6 +19,29 @@ internal class Program {
 
             var logger = loggerFactory.CreateLogger(typeof(Program));
 
+            // Override Newtonsoft.Json MaxDepth de/serialization because
+            // as of version 13.0.1 the default MaxDepth is 64
+            // this keeps pre 13.0.1 behaviour
+            Func<Newtonsoft.Json.JsonSerializerSettings> defaultSettingsFunc =
+               Newtonsoft.Json.JsonConvert.DefaultSettings;
+            if (null == Newtonsoft.Json.JsonConvert.DefaultSettings) {
```

Comment:
Why the need for branching? And why is the NullValueHandling only done in the first branch?

Reply:
I've removed the null values handling override.